source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -235,3 +246,21 @@ void BioXASSideAppController::onCurrentScanActionFinishedImplementation(AMScanAc
 {
 	Q_UNUSED(action)
 }
+
+QGroupBox *BioXASSideAppController::createSqeezeGroupBoxWithView(QString title, QWidget *view)
+{
+	QHBoxLayout *horizontalLayout = new QHBoxLayout;
```

Comment:
@dleacock, this is essentially what I want you to make in that issue I told you about.  There is another one that I'd like to go with it, but the first one is essentially this.